src/app/components/core/logs/system-log/system-log.component.ts

```diff
@@ -233,4 +236,13 @@ export class SystemLogComponent implements OnInit, OnDestroy {
     this.destroy$.unsubscribe();
     this.subscription.unsubscribe();
   }
+
+  getFilteredSysLog() {
```

Comment:
We are using on keyup with delay of 0.3s? 

Add notif service with name "N 1" and north with "N 123" then If I type "N 1" at normal speed for search

- how many requests are there  a
- also try adding 23 to search txt